Gemfile

```diff
@@ -1,14 +1,29 @@
-# If you do not have OpenSSL installed, update
-# the following line to use "http://" instead
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem "middleman", "~>3.3.12"
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+gem "jekyll", "~> 3.8.5"
 
-# Live-reloading plugin
-gem "middleman-livereload", "~> 3.1.0"
+# This is the default theme for new Jekyll sites. You may change this to anything you like.
+#gem "minima", "~> 2.0"
 
-# For faster file watcher updates on Windows:
-gem "wdm", "~> 0.1.0", :platforms => [:mswin, :mingw]
+# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
+# uncomment the line below. To upgrade, run `bundle update github-pages`.
+#gem "github-pages", group: :jekyll_plugins
 
-# Windows does not come with time zone data
-gem "tzinfo-data", platforms: [:mswin, :mingw, :jruby]
\ No newline at end of file
+# If you have any plugins, put them here!
+#group :jekyll_plugins do
+#  gem "jekyll-feed", "~> 0.6"
+#end
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem "tzinfo-data", platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# Performance-booster for watching directories on Windows
+gem "wdm", "~> 0.1.0" if Gem.win_platform?
```

Comment:
Can someone confirm this still works on Windows?  I have had problems with the wdm gem loading and disabled it previously.

Reply:
Personally I've not seen it throw any obvious errors so far, There is a newer version 0.1.1 that fixes an installation issue with ruby 2.2 and has other minor improvement that we could move to.

Reply:
In that case let's switch to 0.1.1

Reply:
Note: the `~>` version specifier tells Bundler to pull in the most recent patch version (in this case 0.1.x), so we should get any fixes until version 0.2.0 whenever we run `bundle update`. That's why @ShmoogleOsukami is seeing 0.1.1 locally with this Gemfile.